src/clients/c++/request.h

```diff
@@ -347,6 +369,21 @@ class InferContext {
     /// model configuration. Variable-size dimensions are reported as
     /// -1.
     virtual const DimsList& Dims() const = 0;
+
+    /// The values for this output are set by reference in a shared memory
```

Comment:
"Indicate that the result values for this output should be placed in a shared memory region instead of being returned in the inference response. The shared memory region must not be modified or..."

I don't think we call this multiple times for batched output, do we? Our entire batch of output must be contiguous in a single region.

Fix param formatting.

Reply:
Single call per batch, thats right.